descwl_shear_sims/simple_sim.py

```diff
@@ -706,10 +706,15 @@ def _psf_render_func(*, x, y, center_psf):
                     x=x-int(x+0.5),
                     y=y-int(y+0.5),
```

Comment:
The offset is here and directly computed from the inputs. No need to send it back out.

Reply:
I want this available so I don't have to hard code `x=x-int(x+0.5)` in other codes that use the offset.  What if conventions change and the offset is calculated differently in the future? etc.

Reply:
Also, it does no harm to have it there.

Reply:
This is the convention for a pixel center. It won't change and hasn't in probably 20+ years. :)

Reply:
Feel free to merge if you want though.

Reply:
A real case where the convention could be different:  what if the stack uses pixel centers that are 0.5 different from galsim?